src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2224,12 +2224,6 @@ struct HeatBalanceData : BaseGlobalStruct
     // (considered diffuse)
     // Originally QD, now used only for EnclSolQSDifSol calc for daylighting
 
-    Array1D<Real64> EnclSolVMULT;        // 1/(Sum Of A Zone's Inside Surfaces Area*Absorptance)
-    Array1D<Real64> EnclRadQThermalRad;  // TOTAL THERMAL RADIATION ADDED TO ZONE or Radiant Enclosure (group of zones)
-    Array1D<Real64> EnclRadThermAbsMult; // EnclRadThermAbsMult  - MULTIPLIER TO COMPUTE 'ITABSF'
-    Array1D<bool> EnclSolAbsFirstCalc;   // for error message
-    Array1D<bool> EnclRadReCalc;         // Enclosure solar or thermal radiation properties needs to be recalc due to window/shading status change
```

Comment:
Moved to enclosure data. Which then required non-substantive changes throughout the code.